docs(s3): update bucket creation with use case MTA-3205 (#2756)
scaleway/docs-content: storage/object/how-to/create-a-bucket.mdx

1. Click **Object Storage** on the left side menu of the console. The Object Storage dashboard displays.
2. Click **+ Create bucket**. The bucket creation page displays.

> 3. Select the geographical location in which your bucket will be created. Scaleway Object Storage is currently available in three Regions.

Reviewer: ```suggestion
3. Select the geographical location in which to create your bucket.  Scaleway Object Storage is currently available in three Regions.
```
